### PR TITLE
[12_5_X] Raw prime wf for Run 3 data w/ dropped strips + approx. clusters

### PR DESCRIPTION
**Pull request by mandrenguyen**
backport of #39863

### CODE REVIEW

**Comment by mandrenguyen** [comment]
please test

**Comment by smorovic** [comment]
+1

**Comment by mandrenguyen** [comment]
please test

**Comment by mandrenguyen** [comment]
please test

**Comment by mandrenguyen** [comment]
please test

**Comment by mandrenguyen** [comment]
urgent
needed for a new release to be part of the heavy-ion test next Monday (november 7)

**Comment by smorovic** [comment]
+daq

File: Configuration/EventContent/python/EventContent_cff.py [hunk 1]
```diff
@@ -633,6 +637,10 @@ def SwapKeepAndDrop(l):
 FEVTDEBUGHLTEventContent.outputCommands.append('keep *_*_MergedTrackTruth_*')
 FEVTDEBUGHLTEventContent.outputCommands.append('keep *_*_StripDigiSimLink_*')
 FEVTDEBUGHLTEventContent.outputCommands.append('keep *_*_PixelDigiSimLink_*')
+approxSiStripClusters.toModify(FEVTDEBUGHLTEventContent,
```
[the remark this answers is not in the document]
Comment (srimanob) [reply Configuration/EventContent/python/EventContent_cff.py:640]: This line in master, 
https://github.com/cms-sw/cmssw/blob/master/Configuration/EventContent/python/EventContent_cff.py#L632

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 2]
```diff
@@ -2078,6 +2082,28 @@ def lhegensim2018ml(fragment,howMuch):
...
+                        '--process':'REHLT'
+}
+
+steps['RECOHID22APPROXCLUSTERS']=merge([{ '--scenario':'pp',
+                                          '--conditions':'auto:run3_data_prompt',
+                                          '-s':'RAW2DIGI,L1Reco,RECO,DQM:@commonFakeHLT+@standardDQMFakeHLT',
+                                          '--datatier':'AOD,DQMIO',
+                                          '--eventcontent':'AOD,DQM',
+                                          '--era':'Run3_pp_on_PbPb_approxSiStripClusters',
+                                          '--repacked':'',
```
Comment (srimanob) [inline Configuration/PyReleaseValidation/python/relval_steps.py:2102]: Just to confirm that this line is different with `RECOHID18APPROXCLUSTERS` (no repack is used),
https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_steps.py#L2083-L2092

**Comment by rappoccio** [comment]
@cms-sw/daq-l2 can you please sign the master PR as well so we can get this in ASAP?

https://github.com/cms-sw/cmssw/pull/39863

**Comment by AdrianoDee** [comment]
+upgrade

**Comment by perrotta** [comment]
@cms-sw/pdmv-l2 please let us know if you have any issue about this backport

**Comment by kskovpen** [comment]
+pdmv

**Comment by perrotta** [comment]
+1

**Pull request merged**